src/transcription/schema.ts

```diff
@@ -0,0 +1,164 @@
+  })
+  .strict()
+
+export const ListTranscriptionsRequestSchema = z
+  .object({
+    organizationName: z.string(),
+  })
+  .strict()
+
+export const TranscribeRequestSchema = z
+  .object({
+    source: z.string().url().describe('URL or file path'),
+    organizationName: z.string(),
+    options: z
+      .object({
+        returnAsFile: z.boolean().optional(),
+        languageCode: z.string().optional(),
+        translate: z.string().optional(),
+        sentenceLevelTimestamps: z.boolean().optional(),
+        wordLevelTimestamps: z.boolean().optional(),
+        diarization: z.boolean().optional(),
+        sentenceDiarization: z.boolean().optional(),
+        srt: z.boolean().optional(),
+        summarize: z.number().optional(),
+        llmTranslation: z.array(z.nativeEnum(TranslationLanguage)).optional(),
+        srtTranslation: z.array(z.nativeEnum(TranslationLanguage)).optional(),
+        customVocabulary: z.string().optional(),
+        customPrompt: z.string().optional(),
+        overallSentimentAnalysis: z.boolean().optional(),
+        overallClassification: z.boolean().optional(),
+      })
+      .transform((data) => ({
+        return_as_file: data.returnAsFile,
+        language_code: data.languageCode,
+        translate: data.translate,
+        sentence_level_timestamps: data.sentenceLevelTimestamps,
+        word_level_timestamps: data.wordLevelTimestamps,
+        diarization: data.diarization,
+        sentence_diarization: data.sentenceDiarization,
+        srt: data.srt,
+        summarize: data.summarize,
+        llm_translation: data.llmTranslation,
+        srt_translation: data.srtTranslation,
+        custom_vocabulary: data.customVocabulary,
+        custom_prompt: data.customPrompt,
+        overall_sentiment_analysis: data.overallSentimentAnalysis,
+        overall_classification: data.overallClassification,
+      }))
+      .optional(),
+    webhookUrl: z.string().url().optional().describe('Webhook URL to receive transcription updates.'),
+  })
+  .strict()
+
+export const TranscribeResponseSchema = z
+  .object({
+    id: z.string(),
+    input: z.object({
+      url: z.string().url(),
+      language_code: z.string().optional(),
+      return_as_file: z.boolean().optional(),
+      word_level_timestamps: z.boolean().optional(),
+      diarization: z.boolean().optional(),
+      sentence_diarization: z.boolean().optional(),
+      srt: z.boolean().optional(),
+      translate: z.string().optional(),
+      llm_translation: z.string().optional(),
+      srt_translation: z.string().optional(),
+      custom_prompt: z.string().optional(),
+      custom_vocabulary: z.string().optional(),
+      sentence_level_timestamps: z.boolean().optional(),
+      summarize: z.number().optional(),
+      overall_classification: z.boolean().optional(),
+      overall_sentiment_analysis: z.boolean().optional(),
+      classification_labels: z.string().optional(),
+    }),
+    inferenceEndpointName: z.string(),
```

Comment:
some of them are camelCased and some snake_cased
Was it intentional?

Reply:
Yes, it's intentional. We transform all fields to camelCase below in code. Response in camelCase is received from salad-cloud-sdk methods